src/css/KeyValue.scss

```diff
@@ -7,8 +7,21 @@
 
   @include time;
 
+  &.key-value-two-rows {
+    height: 40px;
```

Comment:
Can we use scss variables for this or something? For the Desktop version we defined constants for the various common heights, such as the single row height and the two row height: https://github.com/LiveSplit/livesplit-core/blob/master/src/rendering/mod.rs#L108